src/data/reducers/assets.js

```diff
@@ -7,7 +7,8 @@ const initialState = {
   page: 0,
   pageSize: 50,
   assetTypes: {},
-  sort: 'sort',
+  sort: 'date_added',
+  direction: 'desc',
```

Comment:
I figure the initial state of the assetParameters would be a good place to stuff in the default sort state: date_added descending.